src/helpers/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Repository } from "../interfaces";
+
+export function isError(error: unknown): error is Error {
+  return error instanceof Error;
+}
```

Comment:
Why is this required?

Reply:
in the component where I was using the error. It is giving type error. `Object is of type unknown`